server/cache.js

```diff
@@ -17,6 +18,11 @@ module.exports = Cache;
 
 function Cache(ecs){
   Emitter.call(this);
+  this.tasks = LRU({
+    max: 1000000,
+    maxAge: ms('1d'),
```

Comment:
They are immutable we could keep them around forever, this is really just to expire things some point, the cache is also maxed at a million elements so it will a couple of megabytes at most... it seemed fine.